### DISCUSSION TITLE
No volume control for LS49AG95

### DISCUSSION THREAD

**Discussion by AntonioYuen** [MonitorControl/MonitorControl#1334] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

I've tried using monitor control and better display to control the volume of my LS49AG95 (apple silicon). Both display a slider but the volume doesn't change no matter the position of the slider. Brightness however works as expected.

### Steps to reproduce

Change volume on application.

### Expected behavior

Monitor volume should adjust.

### Environment Information (please complete the following information)

```markdown
- macOS version: Ventura 13.2.1
- Mac model: MacBook Pro 14 inch 2021
- MonitorControl version: 4.1.0 Build 7034
- Monitor(s): LS49AG95
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
You are either using HDMI which is not supported in MonitorControl (you can use BetterDisplay for that) or the display does not support DDC or if hardware brightness works then DDC volume.

**Reply by AntonioYuen** [reply]
@waydabber The brightness definitely works. BetterDisplay also has the same issue as monitor control. I am not using HDMI, going through displayport.

That's what I thought was interesting because the brightness control DEFINITELY works. However, the volume control does nothing. So unless DDC can be half implemented on a display, there might be something to look into here.

EDIT:

Ah I might have misread, I think you meant the DDC volume could not be supported.

**Reply by waydabber** [reply]
Yes, sorry, I was not clear. So if hardware DDC works then there is no communication issue, but the display might not implement DDC volume control at firmware level. Many displays are like this.
